Facepunch.Steamworks/Structs/LobbyQuery.cs

```diff
@@ -67,6 +63,87 @@ public LobbyQuery WithKeyValue( string key, string value )
 		}
 		#endregion
 
+		#region Numerical filters
+		internal Dictionary<KeyValuePair<string, int>, LobbyComparison> numericalFilters;
```

Comment:
I don't think Dictionary is the right way to do this. A List with a custom struct would make more sense.